src/helpers/previous-link.js

```diff
@@ -1,11 +1,26 @@
-import domready from 'helpers/domready';
+import domready from './domready';
 
 domready(() => {
-  const previousURL = new URLSearchParams(window.location.search).get('previous');
+  let previousURL;
+  const queryStringPreviousURL = new URLSearchParams(window.location.search).get('previous');
+
+  if (queryStringPreviousURL) {
+    previousURL = queryStringPreviousURL;
+  } else {
+    const savedData = sessionStorage.getItem(window.location.pathname);
+
+    if (savedData) {
+      previousURL = JSON.parse(savedData).previousURL;
+    }
+  }
 
   if (previousURL) {
-    const previousLinks = [...document.querySelectorAll('.js-previous')];
+    const previousLinks = [...document.querySelectorAll('.ons-js-previous')];
```

Comment:
@boxadesign ...but you're already doing it here?

Reply:
That is in error. I've updated that. Only DS classes should contain the prefix.